test_config.py

```diff
@@ -1,13 +1,13 @@
-import sys  # noqa: F401
-
-import pytest  # noqa: F401
-
-
-def check_test_solver_install(solver_class):
-    """Hook called in `test_solver_install`.
-
-    If one solver needs to be skip/xfailed on some
-    particular architecture, call pytest.xfail when
-    detecting the situation.
-    """
-    pass
+import sys  # noqa: F401
+
+import pytest  # noqa: F401
+
+
+def check_test_solver_install(solver_class):
+    """Hook called in `test_solver_install`.
+
+    If one solver needs to be skip/xfailed on some
+    particular architecture, call pytest.xfail when
+    detecting the situation.
+    """
+    pass

```

Comment:
TODO

Reply:
This is not a todo :)
This needs to be modified only if a solver should not be tested.

Reply:
It is for checking if we have all the libraries and tools that we need in solver? Here for now I haven't put any special package in solver.